packages/icons/src/email-input.js

```diff
@@ -0,0 +1,16 @@
+export const EmailInputIcon = () => (
+	<svg
+		width="24"
+		height="24"
+		viewBox="0 0 24 24"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
+	>
+		<path
+			d="M18 6.75H6C5.30964 6.75 4.75 7.30964 4.75 8V16C4.75 16.6904 5.30964 17.25 6 17.25H18C18.6904 17.25 19.25 16.6904 19.25 16V8C19.25 7.30964 18.6904 6.75 18 6.75Z"
+			stroke="#1E1E1E"
```

Comment:
We generally prefer to style these using CSS so the stroke attribute can be omitted here and in the other `<path />` below.